src/js/verifyUser.jsx

```diff
@@ -1,62 +1,52 @@
-import React from "react";
+import React, { Suspense, useEffect } from "react";
 import ReactDOM from "react-dom";
+import { useTranslation } from "react-i18next";
 import { ThemeProvider } from "@emotion/react";
 
 import AccountForm from "./components/AccountForm";
 
-import { getLanguage, jsonRequest } from "./utils";
+import i18n from "./i18n"; // to init localization...
```

Comment:
Looks like this is unused. We can remove it

Reply:
This is used to init the localization, when we import it we run it

Reply:
Ah, I didn't realize that importing `i18n` runs it. Maybe add that to the comment? Something like "// When i18n is imported it's run so keep this import"